docs/pages/branding/about.tsx

```diff
@@ -16,132 +18,191 @@ import BrandingRoot from 'docs/src/modules/branding/BrandingRoot';
 
-export default function Page() {
+function BrandingJoinUs() {
   return (
-    <BrandingRoot>
-      <Container sx={{ mt: [10, 18], mb: [12, 20] }}>
-        <Typography variant="h2">
-          Material-UI is awesome.
-          <br />
-          How can <UnderlinedText>I support the project?</UnderlinedText>
+    <Box
+      sx={{
+        bgcolor: 'secondary.main',
+        color: 'secondary.contrastText',
+        position: 'relative',
+        overflow: 'hidden',
+      }}
+    >
+      <Box
+        component="img"
+        src="/static/branding/block6.svg"
+        alt=""
+        sx={{
+          width: 570,
+          height: 526,
+          position: 'absolute',
+          display: 'block',
+          right: { xs: -360, sm: -350, lg: -300 },
+          bottom: { xs: -300, sm: -200, lg: -80 },
+        }}
+      />
+      <Container
+        maxWidth="md"
+        sx={{
+          py: { xs: 17, sm: 19, md: 20 },
+          display: 'flex',
+          flexDirection: 'column',
+          alignItems: 'center',
+          position: 'relative',
+        }}
+      >
+        <Typography variant="h2" align="center">
+          Join our team of creators &amp; innovators
         </Typography>
-        <Typography sx={{ mt: 2, mb: 9 }}>There are many ways to support Material-UI:</Typography>
-        <Grid container spacing={3}>
-          <Grid item xs={12} md={6}>
-            <BrandingCard icon={<QuestionAnswerIcon fontSize="large" />} title="Spread the word">
-              <Typography sx={{ mt: 2 }}>
-                Evangelize Material-UI by linking to{' '}
-                <Link href="https://material-ui.com">material-ui.com</Link> on your website, every
-                backlink matters! Follow us on{' '}
-                <Link href="https://twitter.com/MaterialUI">Twitter</Link>, like and retweet the
-                important news. Or just talk about us with your friends.
-              </Typography>
-            </BrandingCard>
-          </Grid>
-          <Grid item xs={12} md={6}>
-            <BrandingCard
-              color="info"
-              icon={<FeedbackIcon fontSize="large" />}
-              title="Give us feedback"
-            >
-              <Typography sx={{ mt: 2 }}>
-                Tell us what we&apos;re doing well or where we can improve. Please upvote (
-                <span role="img" aria-label="Thumbs up emoji">
-                  👍
-                </span>
-                ) the issues that you are the most interested in seeing solved.{' '}
-                <Link href="https://github.com/mui-org/material-ui/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc">
-                  Give us feedback
+        <Typography align="center" sx={{ mt: 3, mb: 5 }}>
+          If you love the challenge of doing things differently, empowering creativity, and making
+          real connections along the way–then this may be the place for you.
+        </Typography>
+        <Button
```

Comment:
![image](https://user-images.githubusercontent.com/4512430/105323961-10023180-5bcb-11eb-93c4-9826fe708c1c.png)

Shouldn't the button look like this? Or maybe I am not up to date with the latest version.

Reply:
It's the only white button I could find in the whole design. I went on assuming that it was inconsistent and removed it.

Reply:
Sounds fair :)

Reply:
The blue definitely looks better, and +1 for consistency.

The only feedback I have is how laggy the hover and click styles feel on the buttons.

Reply:
The transition is from the default Material Design styles. It's 250ms. Bootstrap has 150ms. Tailwind 200ms. Chakra 250ms, GitHub (200ms)

I have reduced the color gap, it helps.

Otherwise, we could consider `theme.transitions.duration.shorter` (200ms) instead of `theme.transitions.duration.short` (250ms) for but base button, but it's a large change and I would advocate shouldn't be specific to the branding.